libsplinter/src/auth/oauth/rest_api/resources/callback.rs

```diff
@@ -12,27 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use crate::auth::oauth::UserTokens;
-
 #[derive(Deserialize)]
 pub struct CallbackQuery {
     pub code: String,
     pub state: String,
 }
-
-#[derive(Debug, Serialize, Clone, PartialEq)]
-pub struct CallbackResponse<'a> {
```

Comment:
Couldn't this struct provide a `to_query_params` function?

Reply:
We could just as easily implement that function for the `UserTokens` struct, but I'm not sure how much value that provides since it only ever happens in this endpoint.

Reply:
Yes, but it the behaviour is actix-independent

Reply:
Updated to be a method on `UserTokens`